fdbclient/ManagementAPI.actor.cpp

```diff
@@ -1765,6 +1914,78 @@ ACTOR Future<vector<AddressExclusion>> getExcludedServers(Database cx) {
 	}
 }
 
+ACTOR Future<vector<std::string>> getExcludedLocalities(Transaction* tr) {
+	state RangeResult r = wait(tr->getRange(excludedLocalityKeys, CLIENT_KNOBS->TOO_MANY));
+	ASSERT(!r.more && r.size() < CLIENT_KNOBS->TOO_MANY);
+	state RangeResult r2 = wait(tr->getRange(failedLocalityKeys, CLIENT_KNOBS->TOO_MANY));
+	ASSERT(!r2.more && r2.size() < CLIENT_KNOBS->TOO_MANY);
+
+	vector<std::string> excludedLocalities;
+	for (auto i = r.begin(); i != r.end(); ++i) {
```

Comment:
This looks to me like a ranged-for loop would work. Is there a reason I'm missing to use explicit iterators here?